src/components/v5/common/CompletedAction/CompletedAction.tsx

```diff
@@ -117,6 +120,7 @@ const CompletedAction = ({ action }: CompletedActionProps) => {
       case ColonyActionType.ColonyEditMotion:
       case ColonyActionType.EditExpenditureMotion:
       case ColonyActionType.FundExpenditureMotion:
+      case ExtendedColonyActionType.UpdateColonyObjectiveMotion:
         // @NOTE: Enabling those 2 above temporarily
```

Comment:
Tiny tiny note, but it would be better to have this two lines higher as it now makes the comment incorrect.